docs/using.md

```diff
@@ -1,166 +1,428 @@
+# Getting Started and Building
 
-If running locally you can use the `export` command that Genny provides to export to CSV.
+For build instructions, see the installation guide [here](setup.md).
+
+To try launching Genny, navigate to the root of the Genny repo and run the following: 
+
+    ./run-genny workload src/workloads/docs/HelloWorld.yml
+
+Whenever you have questions about the Genny CLI, you can always use the `-h` option for the top-level Genny CLI or any subcommands:
+
+    ./run-genny -h # See all subcommands
+    ./run-genny workload -h # See args and options for the workload subcommand
+
+
+<a id="org1140a6b"></a>
+
+# Core Concepts
+
+This section introduces the core concepts used by Genny, the minimal required syntax for its inputs, and builds up a basic example.
+
+
+<a id="orgdcd1898"></a>
+
+## What is load generation?
+
+Genny is a **load generation** tool. It is used as a client in [load tests](https://en.wikipedia.org/wiki/Load_testing) to perform a large amount of work against a system under test, measuring the client-side visible responses of said test subject. These measurements may represent things like latency, number of operations performed, how many errors occurred, etc. The procedure Genny uses to perform the work should be as deterministic and repeatable as possible. This procedure is called a **workload**.
+
+Results of a load test can inform developers as to the performance of the test subject in stressful situations. How does a system handle increasing numbers of connections performing conflicting operations? How does a system handle a large number of users performing one operation, then simultaneously switching to another in an instant? How does one user initiating a long-running, expensive operation affect the latencies observed by other users? All these situations can be simulated with Genny.
+
+
+<a id="orgc7904ae"></a>
+
+## What is the system under test?
+
+Genny usually expects to be given a connection string to a system under test. While there's nothing preventing a user from writing a "workload" that spawns the system under test, there is not first-class support for such behavior. Genny is explicitly <span class="underline">not</span> a test orchestrator. For test orchestration needs, consider using [DSI](https://github.com/10gen/dsi/).
+
+By default, Genny will try to connect to a MongoDB server at `localhost:27017`. To pass a MongoDB connection string to Genny, use the `-u` option:
+
+    ./run-genny workload -u [arbitrary_URI] src/workloads/docs/InsertWithNop.yml
+
+For more details on how Genny handles connections, see [Connecting to the Server](#orgd6b0450).
+
+
+<a id="org3610c67"></a>
+
+## What is a workload?
+
+A **workload** is a repeatable procedure that Genny uses to generate load against a system under test. Genny workloads are written in yaml configs that describe how **actors** move through **phases**. This section describes each of these.
+
```

Comment:
Apart from load testing do we do stress testing?

Reply:
Stress testing is mostly just particularly intensive load testing, imho. Added a short note about that.